numba_dppy/tests/passes/test_rename_numpy_function_pass.py

```diff
@@ -94,7 +94,8 @@ def got(a):
 
 
 @pytest.mark.skipif(
-    not ensure_dpnp(), reason="test only when dpnp is available"
+    not ensure_dpnp(),
```

Comment:
Is it okay that there is exist check **ensure_dpnp**?

Reply:
You can create `skip_no_dpnp` mark in helper and reuse it.